packages/utils/telemetry-utils/src/eventEmitterWithErrorHandling.ts

```diff
@@ -12,20 +12,11 @@ import { TypedEventEmitter, EventEmitterEventType } from "@fluidframework/common
  */
 export class EventEmitterWithErrorHandling<TEvent extends IEvent = IEvent> extends TypedEventEmitter<TEvent> {
     public emit(event: EventEmitterEventType, ...args: any[]): boolean {
-        if (event === "error") {
-            const anyListeners = super.emit(event, ...args);
-            if (!anyListeners) {
-                console.error("Nobody is listening for 'error' events");
-            }
-            return anyListeners;
-        }
-
-        let result: boolean;
         try {
-            result = super.emit(event, ...args);
+            return super.emit(event, ...args);
```

Comment:
For future, it would be nice to trace whether the handler is internal version developer so that we can classify them in the logs

Reply:
Really nice idea, I'll open an issue

Reply:
I'm a bit skeptical here, as developer handler may have called into our code that threw an exception. It might be because developer code (usage patter of our API) is wrong, but maybe it's just our bug.
We need analyzers that can take stack and attribute it to who's at fault. I do not know if it's realistic to do in JS world, but this approach works pretty well on native side.